components/form/appearance.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Appearance
+page_title: Form Appearance
+description: Appearance settings of the Form for Blazor.
+slug: form-appearance
+tags: telerik,blazor,form,appearance
+published: True
+position: 35
+---
+
+# Appearance Settings
+
+You can control the appearance of the form by setting the [Size](#size) attribute.
+
+## Size
+
+>The `Size` configuration of the Form affects the padding of all the editors and buttons.
+
+You can increase or decrease the size of the Form by setting the `Size` parameter to a member of the `Telerik.Blazor.ThemeConstants.Form.Size` class:
+
+| Class members | Manual declarations |
+|---------------|--------|
+| `Small`   |`sm`|
+| `Medium`   |`md`|
+| `Large`   |`lg`|
+
+If the size option is not specified, it is set to `Medium` by default. 
```

Comment:
Look how the default options is specified in other appearance articles - e.g. https://docs.telerik.com/blazor-ui/components/autocomplete/appearance#fillmode.

Listed in the table for better visibility. Let's keep that approach for consistency. Applies to all articles.